qemu/qemu_s390x.go

```diff
@@ -0,0 +1,92 @@
+// +build s390x
+
+/*
+// Copyright (c) 2018 Yash Jain
+// Copyright (c) 2016 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+package qemu
+
+// -pci devices don't play well with Z hence corresponding -ccw devices
+// See https://wiki.qemu.org/Documentation/Platforms/S390X
+const (
```

Comment:
I don't understand why we are duplicating all of these constants.  How many of them differ between s390 and non s390.  Is it just the pci devices?  If so we only need these guys to be separated out.